src/elements/content-sidebar/__tests__/DocgenSidebar.test.tsx

```diff
@@ -30,101 +28,63 @@ const defaultProps = {
     ...docGenSidebarProps,
 };
 
-// TODO renable when development returns to normal
-xdescribe('elements/content-sidebar/DocGenSidebar', () => {
-    const getWrapper = (props = defaultProps, options = {}) =>
-        mount(<DocGenSidebar logger={{ onReadyMetric: jest.fn() }} {...props} />, options);
+describe('elements/content-sidebar/DocGenSidebar', () => {
+    const renderComponent = (props = defaultProps) =>
+        render(<DocGenSidebar logger={{ onReadyMetric: jest.fn() }} {...props} />);
 
     test('componentDidMount() should call fetch tags', async () => {
-        await act(async () => {
-            await getWrapper(defaultProps);
-        });
-        expect(docGenSidebarProps.getDocGenTags).toHaveBeenCalled();
+        renderComponent();
+        await waitFor(() => expect(docGenSidebarProps.getDocGenTags).toHaveBeenCalled());
     });
 
     test('should render DocGen sidebar component correctly with tags list', async () => {
-        let wrapper;
-        await act(async () => {
-            wrapper = getWrapper(defaultProps);
-        });
-        wrapper!.update();
-        const tagList = wrapper!.find('span.bcs-DocGen-tagPath');
-        expect(tagList).toHaveLength(26);
-        expect(wrapper).toMatchSnapshot();
+        renderComponent();
+        const tagList = await screen.findAllByTestId('bcs-TagsSection');
+        expect(tagList).toHaveLength(2);
     });
 
     test('should render empty state when there are no tags', async () => {
-        let wrapper;
-        await act(async () => {
-            wrapper = await getWrapper({
-                ...defaultProps,
-                getDocGenTags: noTagsMock,
-            });
+        renderComponent({
+            ...defaultProps,
+            getDocGenTags: noTagsMock,
         });
 
-        wrapper!.update();
-        const emptyState = wrapper!.find(FormattedMessage).at(0);
-        expect(emptyState.prop('defaultMessage')).toEqual('This document has no tags');
-        expect(emptyState).toHaveLength(1);
-        expect(wrapper).toMatchSnapshot();
+        const emptyState = await screen.findByText('This document has no tags');
+        expect(emptyState).toBeInTheDocument();
     });
 
     test('should render loading state', async () => {
-        const wrapper = await getWrapper({
+        renderComponent({
             ...defaultProps,
             getDocGenTags: noTagsMock,
         });
-        const loadingState = wrapper!.find(LoadingIndicator);
-        expect(loadingState).toHaveLength(1);
-        expect(wrapper).toMatchSnapshot();
+
+        const loadingState = await screen.getByRole('status'); // Assuming LoadingIndicator has a test id
+        expect(loadingState).toBeInTheDocument();
```

Comment:
```suggestion
        expect(screen.getByRole('status', { name: 'loading' })).toBeInTheDocument();
```
when you update the aria-label to the common translated loading message you can be more specific on the targetting